README.md

```diff
@@ -55,7 +55,7 @@ You can configure `rofimoji` either with cli arguments or with a config file cal
 |-------------------------------------------------------------------------------------------------------------------------|--------------|----------------------------------------------------------------------------------------------------------------|---------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `--action`                                                                                                              | `-a`         | `type`, `copy`, `clipboard`, `unicode`, `copy-unicode`, `print`                                                | `type`                                      | Chose what `rofimoji` should do with the selected characters. See [Insertion Method](#insertion-method).<br/>`type`: Directly type the characters into the last active window.<br/>`copy`: Copy them to the clipboard.<br/>`clipboard`: Insert the selected characters through pasting from the clipboard, instead of directly typing them.<br/>`unicode`: Type the unicode codepoints of the selected characters.<br/>`copy-unicode`: Copy the codepoints to clipboard.<br/>`print`: Print the chosen characters to `stdout`. |
+| `--action`                                                                                                              | `-a`         | `type`, `copy`, `clipboard`, `unicode`, `copy-unicode`, `print`                                                | `type`                                      | Choose what `rofimoji` should do with the selected characters. Multiple actions can be specified. See [Insertion Method](#insertion-method).<br/>`type`: Directly type the characters into the last active window.<br/>`copy`: Copy them to the clipboard.<br/>`clipboard`: Insert the selected characters through pasting from the clipboard, instead of directly typing them.<br/>`unicode`: Type the unicode codepoints of the selected characters.<br/>`copy-unicode`: Copy the codepoints to clipboard.<br/>`print`: Print the chosen characters to `stdout`. |
```

Comment:
I'd like to rephrase this a bit, to make it both more specific and a better flow (hopefully 😉):
```
Choose what `rofimoji` should do with the selected characters. Multiple actions can be specified with a space character in between (f.e. `-a type copy`).<br />
See also [Insertion Method](#insertion-method).<br/>
[...]
```